packages/data-broker/src/services/tupaia/TupaiaService.ts

```diff
@@ -0,0 +1,159 @@
+/**
```

Comment:
Looks like git is pretty smart and can track most of these renames, but not all. It would be nice to keep the history for git blame, and to do that we would need to split up this commit, one to rename the files with no changes, and the second to add the types. 

I think it's still worth doing and not too hard to split that up. If you do a new commit with just the renames and then cherry-pick this commit on top that should do the trick.

Reply:
Aah that's a good trick, thanks for bringing it to my attention!!

Reply:
@IgorNadj I tried that and it works for individual commits, however those files still show separately in overall diff in GitHub....

It is a pity because it makes reviewing harder